### PR TITLE
[node] Use Volta for NodeJS and yarn versioning

### PR DESCRIPTION
**Pull request by lefebvree**
### What and why?

- Update the project to use [Volta](https://volta.sh/) for NodeJS and yarn versioning.
- Update some development related sections in the readme.

### How?

The node version is now committed in the `package.json` and not `.node-version` anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,11 +66,10 @@ yarn launch synthetics run-tests --config dev/global.config.json
 
 ### Framework and libraries used
 
-This tool uses [clipanion](https://github.com/arcanis/clipanion) to handle the different commands.
-
-The tests are written using [jest](https://github.com/facebook/jest).
-
-The coding style is checked with [tslint](https://github.com/palantir/tslint) and the configuration can be found in the [tslint.json](/tslint.json) file.
+- [clipanion](https://github.com/arcanis/clipanion): CLI library to handle the different commands
```
Comment (drichards-87) [inline README.md:69]:
```suggestion
- [clipanion](https://github.com/arcanis/clipanion): CLI library to handle the different commands.
```

File: README.md [hunk 2]
```diff
@@ -66,11 +66,10 @@ yarn launch synthetics run-tests --config dev/global.config.json
...
+- [eslint](https://github.com/eslint/eslint): Linting ([.eslintrc.js](/.eslintrc.js)).
+- [jest](https://github.com/facebook/jest): Testing
+- [volta](https://github.com/volta-cli/volta): NodeJS and yarn versioning
```
Comment (drichards-87) [inline README.md:72]:
```suggestion
- [volta](https://github.com/volta-cli/volta): NodeJS and yarn versioning.
```

File: README.md [hunk 3]
```diff
@@ -66,11 +66,10 @@ yarn launch synthetics run-tests --config dev/global.config.json
...
 ### Framework and libraries used
 
-This tool uses [clipanion](https://github.com/arcanis/clipanion) to handle the different commands.
-
-The tests are written using [jest](https://github.com/facebook/jest).
-
-The coding style is checked with [tslint](https://github.com/palantir/tslint) and the configuration can be found in the [tslint.json](/tslint.json) file.
+- [clipanion](https://github.com/arcanis/clipanion): CLI library to handle the different commands
+- [eslint](https://github.com/eslint/eslint): Linting ([.eslintrc.js](/.eslintrc.js)).
+- [jest](https://github.com/facebook/jest): Testing
```
Comment (drichards-87) [inline README.md:71]:
```suggestion
- [jest](https://github.com/facebook/jest): Tests are written in Jest.
```

**Review by drichards-87 — APPROVED** [review]
Added some minor feedback from Docs. PR is approved.

**Review by juan-fernandez — APPROVED** [review]

**Pull request merged**